packages/workspace/src/browser/workspace-service.ts

```diff
@@ -32,8 +31,7 @@ import { WorkspacePreferences } from './workspace-preferences';
 @injectable()
 export class WorkspaceService implements FrontendApplicationContribution {
 
-    // TODO remove it with the patch where the config file becomes independent from the workspace
-    private _workspaceFolder: FileStat | undefined;
+    private _workspaceMetaDataFile: FileStat | undefined;
```

Comment:
I would just call it  `_workspaceFile`, it is already clear that it contains workspace metadata and then get rid of `metadata` in other variable and method names.